skimage/measure/_moments.py

```diff
@@ -445,4 +445,11 @@ def inertia_tensor_eigvals(image, mu=None, T=None):
     if T is None:
         T = inertia_tensor(image, mu)
     eigvals = np.linalg.eigvalsh(T)
+    """
+    Floating point precision problems could make a positive
+    semidefinite matrix have an eigenvalue that is very slightly
+    negative.  This can cause problems down the line, so set values
+    very near zero to zero.
+    """
+    eigvals = np.where(abs(eigvals) < 1e-12, 0, eigvals)
```

Comment:
Rather than clamping, would it not be better to clip negative eigenvalues altogether? ie `eigvals = np.clip(eigvals, 0, None)`. I don't see why `+1e-13` should be set to 0?

Reply:
```suggestion
    eigvals = np.clip(eigvals, 0, None, out=eigvals)
```

Reply:
In theory, someone could supply a mu or T argument that was deliberately not originated from a positive semidefinite matrix.  In that case, significantly negative values in the response would be justified.  Whether someone is actually abusing the code in that way, I don't  know.

Reply:
Does usng the out=eigvals argument make the assignment to eigvals redundant?